src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.Converters.cs

```diff
@@ -25,6 +26,7 @@ public sealed partial class JsonSerializerOptions
         // The cached converters (custom or built-in).
         private readonly ConcurrentDictionary<Type, JsonConverter?> _converters = new ConcurrentDictionary<Type, JsonConverter?>();
 
+        [RequiresUnreferencedCode(JsonSerializer.SerializationUnreferencedCodeMessage)]
```

Comment:
Without `[RequiresUnreferencedCode]` does this produce a warning or error?

Reply:
(if not, wondering how we'd know to add it)

Reply:
> Without [RequiresUnreferencedCode] does this produce a warning or error?

It produces an ILLink warning because this method calls `new IEnumerableConverterFactory()` and `new ObjectConverterFactory()`. The constructors of those classes are marked `RequiresUnreferencedCode`.